src/main/java/core/basesyntax/service/impl/operation/ReturnOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+
+public class ReturnOperationHandler implements OperationHandler {
+
+    @Override
+    public void handle(FruitTransaction transaction) {
+        new SupplyOperationHandler().handle(transaction);
```

Comment:
do not create new SupplyOperationHandler() inside method.
Make it class variable or use inheritance and super.handle()